wordpress-mobile/WordPress-Android
# Issues/6542 track prologue swipes

Fixes #6542 

To test:
Set a break point where the stat is being bumped.
View the prologue and swipe through the screens. 
Confirm the stat is being bumped.
